### PR TITLE
update q2-sample-classifier citation

### PR DESCRIPTION
**Pull request by nbokulich**

### CODE REVIEW

File: q2_sample_classifier/citations.bib [hunk 1]
```diff
@@ -2,8 +2,11 @@ @article {Bokulich306167
 	author = {Bokulich, Nicholas and Dillon, Matthew and Bolyen, Evan and Kaehler, Benjamin D and Huttley, Gavin A and Caporaso, J Gregory},
 	title = {q2-sample-classifier: machine-learning tools for microbiome classification and regression},
```
Comment (thermokarst) [inline q2_sample_classifier/citations.bib]:
```suggestion
	title = {{q2-sample-classifier}: machine-learning tools for microbiome classification and regression},
```

This should fix https://github.com/qiime2/docs/issues/341, but would you mind double-checking @nbokulich ?

**Comment by nbokulich** [comment]
thanks for committing that change @thermokarst ! LGTM.

**Comment by thermokarst** [comment]
Did that change work for you, @nbokulich? I am still seeing a capital Q, and, when I check in q2cli it looks like those extra curly braces have been stripped. @ebolyen, does the citations API do some sanitizing when loading bibtex entries?

**Comment by nbokulich** [comment]
> Did that change work for you

I did not test... you sounded so sure so I just assumed your fix would work! I am seeing the same thing: the extra curly braces are missing when I check in the CLI.

> all of your new lines have tabs

Nick's nits picked, see latest commit

**Comment by ebolyen** [comment]
Re: bibtex, it is technically not using your bibtex, but reading it to generate an object for that entry, and then it re-writes it as needed. That said, the curly braces *should* have worked... Perhaps the Biber engine has stronger rules on title-casing?

**Comment by thermokarst** [comment]
> it is technically not using your bibtex

Yep, I could've been more clear with my question, I wanted to know what kinds of transformations were being applied to the bibtex entry.

Based on my poking around, the offending config is [this little bit](https://github.com/qiime2/qiime2/blob/master/qiime2/core/cite.py#L27-L29). Looks like this is an [outstanding TODO](https://github.com/sciunto-org/python-bibtexparser/blob/master/bibtexparser/latexenc.py#L70-L72) in the parser.

**Comment by thermokarst** [comment]
@nbokulich, confirmed that the rendering of the title works fine when the parser stops stripping braces from the entries. Will merge as-is, but nothing will change on the docs side of things until the parser dependency is updated (or, we come up with another way to unicodify entries).

**Comment by thermokarst** [comment]
@nbokulich - can you add a note to the changelog?

**Pull request merged**